packages/core-sdk/src/resources/royalty.ts

```diff
@@ -334,8 +339,6 @@ export class RoyaltyClient {
 
   /**
    * Get the royalty vault proxy address of given royaltyVaultIpId.
-   * @param royaltyVaultIpId the id of the royalty vault.
```

Comment:
just a small note that this `@param` does not conflict and can be kept. But i don't think the description is needed as the variable name already tell us this, so good to remove.